pkg/routeagent/controllers/route/route.go

```diff
@@ -256,14 +253,12 @@ func (r *RouteController) handleRemovedEndpoint(obj interface{}) {
 	if object, ok = obj.(*v1.Endpoint); !ok {
 		tombstone, ok := obj.(cache.DeletedFinalStateUnknown)
 		if !ok {
-			utilruntime.HandleError(fmt.Errorf("error decoding object, invalid type"))
-			klog.V(4).Infof("problem decoding object")
+			klog.Errorf("Error decoding object %v, invalid type", obj)
 			return
 		}
 		object, ok = tombstone.Obj.(*v1.Endpoint)
 		if !ok {
-			utilruntime.HandleError(fmt.Errorf("error decoding object tombstone, invalid type"))
-			klog.V(4).Infof("problem decoding object tombstone")
+			klog.Errorf("Error decoding object tombstone %v, invalid type", tombstone.Obj)
```

Comment:
To be in line with #28 
```suggestion
			klog.Errorf("Could not convert object tombstone %v to an Endpoint", tombstone.Obj)
```